toolbox/Xamarin.Forms.toolbox.xml

```diff
@@ -0,0 +1,167 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<Catalog Version="3.3">
+	<Category Name="Controls">
+		<Item Class="Xamarin.Forms.BoxView" Name="BoxView" Image="view-BoxView-16.png">
+			<Snippet>
+				<BoxView />
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.ActivityIndicator" Name="ActivityIndicator" Image="view-ActivityIndicator-16.png">
+			<Snippet>
+				<ActivityIndicator />
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.Button" Name="Button" Image="view-Button-16.png">
+			<Snippet>
+				<Button Text="" />
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.DatePicker" Name="DatePicker" Image="view-DatePicker-16.png">
+			<Snippet>
+				<DatePicker />
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.Editor" Name="Editor" Image="view-Editor-16.png">
+			<Snippet>
+				<Editor />
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.Entry" Name="Entry" Image="view-Entry-16.png">
+			<Snippet>
+				<Entry Placeholder="" />
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.Image" Name="Image" Image="view-Image-16.png">
+			<Snippet>
+				<Image Source="" />
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.ListView" Name="ListView" Image="view-ListView-16.png">
+			<Snippet>
+				<ListView ItemsSource="">
+					<ListView.ItemTemplate>
+						<DataTemplate>
+						</DataTemplate>
```

Comment:
why not going one step further and adding the `ViewCell` ?

Reply:
Snippets provided by PM